recipes-graphics/mesa/mesa_%.bbappend

```diff
@@ -1,6 +1,3 @@
-LDFLAGS:append:toolchain-clang = " -latomic -lm"
-DEPENDS:append:toolchain-clang = " libatomic-ops"
```

Comment:
I think the real test is mips and riscv32 or ppc32 where 64bit builtin atomics are missing

Reply:
`MACHINE=qemumips bitbake mesa-demos` just build successfully. Is that enough?

Reply:
seems good thanks